package/toltec-meta/package

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+archs=(rm1 rm2)
+pkgnames=(toltec-meta)
```

Comment:
Since this describes the base set of packages for Toltec, how about `toltec-base` as a name? `toltec-meta` is correct but seems a bit too generic to me, as a meta package can really be about anything.